extension/src/debugger/AspireDebugConfigurationProvider.ts

```diff
@@ -26,7 +25,7 @@ export class AspireDebugConfigurationProvider implements vscode.DebugConfigurati
         });
 
         try {
-            for (const candidate of await this.computeAppHostCandidates(folder)) {
+            for (const candidate of await this._availableProjectsService.getAppHostCandidates(folder)) {
```

Comment:
It seems like `computeAppHostCandidates` used to throw which would trigger a warning message to the user. Now, all the error handling is encapsulated in the AvailableProjectService and we won't actually get an error surfacing here (and hence no warning to the user). Is that correct/expected?